diff --git a/dist/init-D1800OqA.mjs b/dist/init-D1800OqA.mjs
index c3167954ca4adf019742b71c56eb7d73abf41a86..d789376dfb3d84893c345ba6829874a4291b88a4 100644
--- a/dist/init-D1800OqA.mjs
+++ b/dist/init-D1800OqA.mjs
@@ -293,6 +293,9 @@ var AddonsApi = class {
 	addonVersionCreateEndpoint(extensionId) {
 		return new URL(`https://addons.mozilla.org/api/v5/addons/addon/${extensionId}/versions/`);
 	}
+	addonVersionEditEndpoint(extensionId, versionId) {
+		return new URL(`https://addons.mozilla.org/api/v5/addons/addon/${extensionId}/versions/${versionId}`)
+	}
 	/**
 	* Docs: https://addons-server.readthedocs.io/en/latest/topics/api/addons.html#detail
 	*/
@@ -330,7 +333,7 @@ var AddonsApi = class {
 		if (params.sourceFile) form.set("source", await fileFromPath(params.sourceFile));
 		else form.set("source", "");
 		const encoder = new FormDataEncoder(form);
-		return await fetch(endpoint.href, {
+		const createResponse = await fetch(endpoint.href, {
 			method: "POST",
 			body: Readable.from(encoder),
 			headers: {
@@ -338,6 +341,18 @@ var AddonsApi = class {
 				Authorization: this.getAuthHeader()
 			}
 		});
+		const patchEndpoint = this.addonVersionEditEndpoint(params.extensionId, createResponse.id);
+		await fetch(patchEndpoint.href, {
+			method: "PATCH",
+			body: JSON.stringify({
+				compatibility: ['firefox', 'android']
+			}),
+			headers: {
+				Authorization: this.getAuthHeader(),
+				"Content-Type": "application/json"
+			}
+		});
+		return createResponse;
 	}
 	/**
 	* See https://addons-server.readthedocs.io/en/latest/topics/api/auth.html
